models/tokens/optimism/tokens_optimism_erc20_curated.sql

```diff
@@ -394,6 +394,7 @@ WITH raw_token_list AS (
     ,(0x3a18dcc9745edcd1ef33ecb93b0b6eba5671e7ca, 'KUJI', 6,  'underlying')
     ,(0xdC6fF44d5d932Cbd77B52E5612Ba0529DC6226F1, 'WLD', 18, 'underlying')
     ,(0x894134a25a5faC1c2C26F1d8fBf05111a3CB9487, 'GRAI', 18, 'underlying')
+    ,(0x50Bce64397C75488465253c0A034b8097FeA6578, 'HAN', 18, 'underlying')
```

Comment:
This address is already added:
https://github.com/duneanalytics/spellbook/blob/35a30c19b2ee2985d344fb6b8a104a8a123f8c8c/models/tokens/optimism/tokens_optimism_erc20_curated.sql#L245

Reply:
ah yeah merge error. resolving!